rasa/shared/importers/importer.py

```diff
@@ -48,16 +48,23 @@ async def get_stories(
         Returns:
             `StoryGraph` containing all loaded stories.
         """
-
+        # TODO: Drop `use_e2e` in Rasa Open Source 3.0.0 when removing Markdown support
         raise NotImplementedError()
 
+    async def get_conversation_tests(self) -> StoryGraph:
+        """Retrieves end-to-end conversation stories for testing.
+
+        Returns:
+            `StoryGraph` containing all loaded stories.
+        """
+        return await self.get_stories(use_e2e=True)
```

Comment:
This is implemented as fallback. Ideally they override this when they have a custom `TrainingDataImporter` class. We cannot let it raise in case it's not implemented as this might break existing implementations of custom `TrainingDataImporter`s